libraries/botbuilder-core/botbuilder/core/__init__.py

```diff
@@ -20,22 +20,29 @@
 from .test_adapter import TestAdapter
 from .user_state import UserState
 
-__all__ = ['AnonymousReceiveMiddleware',
-           'BotAdapter',
-           'BotContext',
-           'BotFrameworkAdapter',
-           'BotFrameworkAdapterSettings',
-           'BotState',
-           'calculate_change_hash',
-           'CardFactory',
-           'ConversationState',
-           'MemoryStorage',
-           'MessageFactory',
-           'Middleware',
-           'MiddlewareSet',
-           'Storage',
-           'StorageKeyFactory',
-           'StoreItem',
-           'TestAdapter',
-           'UserState',
-           '__version__',]
+"""Expose botbuilder-schema in botbuilder-core."""
+from botbuilder.schema import *
+import botbuilder.schema
+
+__all__ = [class_name for class_name in dir(botbuilder.schema) if class_name[0].isupper()]
```

Comment:
As we discussed we'll stop mirroring botbuilder-js for this point; `botframework-schema`'s exposure in [`botbuilder-core`](https://github.com/Microsoft/botbuilder-js/blob/6859e19d0444e1d8188e04db403de5fedb77c0a4/libraries/botbuilder-core/src/index.ts#L12) and therefore [`botbuilder`](https://github.com/Microsoft/botbuilder-js/blob/6859e19d0444e1d8188e04db403de5fedb77c0a4/libraries/botbuilder/src/index.ts#L9) was just a convenience item, so that people don't have to import multiple packages manually to use everything available in the upper-level packages (botbuilder* vs botframework-schema and botframework-connector). 

Going forward we can follow the js pattern if people think its convenience is greater than having to keep track of all packages they need to import.